### PR TITLE
Candidate results headers

### PR DESCRIPTION
**Pull request by Jbur43**
This is so the Results in Dash Profiles have section headers and for the Widgets match what customers see on an Assessment Results page

### CODE REVIEW

**Comment by tomprats** [comment]
@Jbur43 This change might not doable as-is. If our customers using the widgets have their own headings, this would add an extra one. I'd either check with @efleming that it'd be fine, or we could just add an option to add the heading conditionally. Like `allowHeadings` with a false default

File: src/components/results/candidate-results/personality-dimension-details/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import PropTypes from "prop-types";
+import {Component} from "react";
+import {detailWithPerspective} from "lib/helpers";
+import {rgba} from "lib/helpers/color";
+import TraitifyPropTypes from "lib/helpers/prop-types";
+import withTraitify from "lib/with-traitify";
+import DetailsList from "./details-list";
+import style from "./style";
+
+class PersonalityDimensionDetails extends Component {
```
Comment (tomprats) [inline src/components/results/candidate-results/personality-dimension-details/index.js:10]: I think this component was folded over into components/results/personality/dimension/details

File: test/components/results/__snapshots__/candidate-results.test.js.snap [hunk 2]
```diff
@@ -2,21 +2,85 @@
...
+        }
+      }
+    />
+     
+    personality_details, [object Object]
+  </p>
   <div
     className="mock"
   >
+<<<<<<< HEAD
```
Comment (tomprats) [inline test/components/results/__snapshots__/candidate-results.test.js.snap:35]: Some merge errors

File: test/components/results/__snapshots__/candidate-results.test.js.snap [hunk 3]
```diff
@@ -27,26 +91,107 @@ exports[`CandidateResults renders results in firstPerson 1`] = `
...
```
Comment (tomprats) [inline test/components/results/__snapshots__/candidate-results.test.js.snap:124]: Some merge errors

File: test/components/results/__snapshots__/candidate-results.test.js.snap [hunk 4]
```diff
@@ -27,26 +91,107 @@ exports[`CandidateResults renders results in firstPerson 1`] = `
...
+        }
+      }
+    />
+     
+    personality_breakdown, [object Object]
+  </p>
   <div
     className="mock"
   >
+<<<<<<< HEAD
```
Comment (tomprats) [inline test/components/results/__snapshots__/candidate-results.test.js.snap:162]: Some merge errors

File: test/components/results/personality/dimension/__snapshots__/details.test.js.snap [hunk 5]
```diff
@@ -1,482 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`PersonalityDimensionDetails renders component 1`] = `
```
Comment (tomprats) [inline test/components/results/personality/dimension/__snapshots__/details.test.js.snap:3]: I think this should still exist

**Review by tomprats — CHANGES_REQUESTED** [review]

**Comment by Jbur43** [comment]
Sorry meant for that to be a WIP commit

**Review by tomprats — COMMENTED** [review]
I think it looks great but two things:

1. Do we need to have it for more of the views? Looks like this is just the candidate view
2. Could we also have a disableHeaders function. Just good to have both options, even if it does disable by default

**Comment by Jbur43** [comment]
Spoke with Eric and ended up adding them for Type Based Results. Also added the disableHeaders function

File: src/lib/i18n-data/ja_jp.json [hunk 6]
```diff
@@ -46,8 +47,14 @@
   "no_careers": "キャリアなし",
   "no_more_careers": "これ以上のキャリア",
   "not_me": "私ではない",
+  "personality_advice": "人格アドバイス",
+  "personality_breakdown": "性格内訳",
+  "personality_blend": "",
```
Comment (tomprats) [inline src/lib/i18n-data/ja_jp.json:52]: I think we want a value here

File: src/lib/i18n-data/ht_us.json [hunk 7]
```diff
@@ -46,8 +47,14 @@
   "no_careers": "Pa gen karyè",
   "no_more_careers": "Pa gen plis karyè",
   "not_me": "Se Pa Mwen",
+  "personality_advice": "Konsèy Pèsonalize",
+  "personality_blend": "",
```
Comment (tomprats) [inline src/lib/i18n-data/ht_us.json:51]: And here

**Review by tomprats — COMMENTED** [review]
I think it's just missing two translation values but everything else looks great!

**Review by tomprats — APPROVED** [review]

**Pull request merged**